src/lib/isMobilePhone.js

```diff
@@ -68,6 +68,7 @@ const phones = {
   'es-PA': /^(\+?507)\d{7,8}$/,
   'es-PY': /^(\+?595|0)9[9876]\d{7}$/,
   'es-UY': /^(\+598|0)9[1-9][\d]{6}$/,
+  'es-VE': /^(\+?58)?((2|4)(\d{2}))(\d{7})$/,
```

Comment:
```suggestion
  'es-VE': /^(\+?58)?(2|4)\d{9}$/,
```
Isn't this the same?

Reply:
Hi @fedeci . Thanks for reviewing it. 

You are right, both regular expressions recognize the same patterns. I preferred to use the first one because the structure is closer to the Venezuela, phone number structure that makes it easy to read/modify. But I am open to suggestions

Reply:
Even if the structure is easier to read, it's adding an unnecessary step to the regex validation